### PR TITLE
Added nrnh5 package

### PR DESCRIPTION
**Pull request by ohm314**
A little spack package for nrnh5

### CODE REVIEW

File: var/spack/repos/builtin/packages/nrnh5/package.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+import os
```
Comment (pramodk) [inline var/spack/repos/builtin/packages/nrnh5/package.py]: os not needed?

File: var/spack/repos/builtin/packages/nrnh5/package.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    """Neuron HDF5 library developed by Blue Brain Project, EPFL"""
+
+    homepage = "https://bbpcode.epfl.ch/sim/nrnh5"
+    url      = "ssh://bbpcode.epfl.ch/sim/nrnh5"
+
+    version('develop', git=url, preferred=True)
+
+    variant('tests', default=False, description="Build unit tests")
+
+    depends_on('cmake@3.5:', type='build')
```
Comment (pramodk) [inline var/spack/repos/builtin/packages/nrnh5/package.py]: I think this could be lower? 3.1?

File: sysconfig/bb5/users/modules.yaml [hunk 3]
```diff
@@ -35,6 +35,7 @@ modules:
       - functionalizer
       - git
       - neurodamus
+      - nrnh5
```
Comment (pramodk) [inline sysconfig/bb5/users/modules.yaml]: remove this and use `spack location -i nrnh5` to get path.

File: var/spack/repos/builtin/packages/nrnh5/package.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/nrnh5/package.py:25]: Not that it matters much, but the license text is outdated by now :)

**Review by matz-e — APPROVED** [review]
LGTM

**Pull request merged**